src/tests/conformance/query/ping_pong.rs

```diff
@@ -41,3 +49,102 @@ async fn c003_TM_PING_expect_pong() {
     perform_expected_message_test(TestConfig::default().with_initial_message(payload), &check)
         .await;
 }
+
+#[tokio::test]
+#[allow(non_snake_case)]
+async fn c003_t2_TM_PING_expect_ping() {
+    // ZG-CONFORMANCE-003
+
+    // Create a rippled node.
+    let target = TempDir::new().expect("unable to create TempDir");
+    let mut node = Node::builder()
+        .start(target.path(), NodeType::Stateful)
+        .await
+        .expect("unable to start the rippled node");
+
+    // Create a synthetic node and connect it to rippled.
```

Comment:
```rust
    // Create a synthetic node and connect it to the rippled.
```
or we can prefer an even more generic one (so later copy/pasting similar tests between repos is easier):
```rust
    // Create a synthetic node and connect it to the node.
```

Reply:
Done